### ISSUE TITLE
Create Euyor's Theme Phase 1

### ISSUE THREAD

**Issue by Pseudo-64** [Team-Awkward-Smile/Epilogue#96]
Euyor as a boss appears out of nowhere, as he breaks through the background like Kool-Aid man. So, it is supposed to be this panicky metal track, where you feel like the musician having a panic attack on the instruments, as if the track maker themselves is trying to understand where the cinnamon-toast fuck this guy appeared from. The ending tunes of phase 1 should lead to a smooth transition with opening tunes of Phase 2, while also have their own loop.

So basically imagine, how would you write an entry + boss fight theme for Kool Aid man if he was a horror monster. 

<img width="360" height="247" alt="Image" src="https://github.com/user-attachments/assets/fd9c6785-ac9e-49b1-a1dc-efa4d18e7600" />

Also if it helps, here's the concept art for Euyor: 

<img width="800" height="600" alt="Image" src="https://github.com/user-attachments/assets/c331b96e-86a4-40e8-8e7a-be9440e7341f" />
